scripts/rotating_camera/worlds/rotating_camera.wbt

```diff
@@ -0,0 +1,28 @@
+#VRML_SIM R2020a utf8
+WorldInfo {
+  info [
+    "Example use of a Supervisor to turn around a robot keeping the ViewPoint oriented toward him."
```

Comment:
```suggestion
    "Example of a supervisor controller rotating the Viewpoint to around a Robot, so that the Robot is always in the field of view."
```